assets/app/scripts/controllers/pod.js

```diff
@@ -47,6 +47,35 @@ angular.module('openshiftConsole')
       $scope.logCanRun = !(_.includes(['New', 'Pending', 'Unknown'], pod.status.phase));
     };
 
+    var setContainerVars = function() {
+      if(!$scope.pod) {
+        return;
+      }
+      var containerStatus = _.find($scope.pod.status.containerStatuses, { name: $scope.logOptions.container });
+      var state = _.get(containerStatus, 'state');
```

Comment:
I think I'd rather just handle `!containerStatus` than use `_.get` here and below. `_.get` seems odd to get a non-nested property.